kiva/pdfmetrics.py

```diff
@@ -418,18 +423,17 @@ def _loadMetrics(self, afmFileName):
         self.stemV = topLevel.get('stemV', 0)
         self.xHeight = topLevel.get('XHeight', 1000)
 
-        strBbox = topLevel.get('FontBBox', [0,0,1000,1000])
-        tokens = string.split(strBbox)
-        self.bbox = []
-        for tok in tokens:
-            self.bbox.append(string.atoi(tok))
+        try:
+            strBbox = topLevel['FontBBox']
+            self.bbox = [int(tok) for tok in strBbox.split()]
+        except KeyError:
+            self.bbox = [0,0,1000,1000]
```

Comment:
It just doesn't work in python 2.7. You would get a `AttributeError`. I don't know how you can keep the structure from before. This is the best I can come up with while respecting the intent.